curriculum/challenges/english/02-javascript-algorithms-and-data-structures/debugging/prevent-infinite-loops-with-a-valid-terminal-condition.english.md

```diff
@@ -54,6 +54,11 @@ function myFunc() {
 <section id='solution'>
 
 ```js
-// solution required
+function myFunc() {
+ for (let i = 1; i <= 4; i += 2) {
+   console.log("Still going!");
+ }
+}
+
```

Comment:
```suggestion
```
Remove this line please

Reply:
I'm making this small change for you, hope you don't mind

Reply:
I have been getting that error on almost all PRs even though the check mark is on `Allow edits from maintainers.`